doc/neps/nep-0048-spending-project-funds.rst

```diff
@@ -0,0 +1,454 @@
+  activity.
+- Principles for spending funds that are intended for NumPy development, but
+  don't fall in the category of NumPy unrestricted funds. This includes most of
+  the grant funding, which is usually earmarked for certain
+  activities/deliverables and goes to an Institutional Partner rather than
+  directly to the NumPy project, and companies or institutions funding specific
+  features.
+  *Rationale: As a project, we have no direct control over how this work gets
+  executed (at least formally, until issues or PRs show up). In some cases, we
+  may not even know the contributions were funded or done by an employee on
+  work time. (Whether that's the case or not should not change how we approach
+  a contribution).  For grants though, we do expect the research/project leader
+  and funded team to align their work with the needs of NumPy and be
+  receptive to feedback from other NumPy maintainers and contributors.*
+
+
+Principles of spending project funds
+------------------------------------
+
+NumPy will likely always be a project with many times more volunteer
+contributors than funded people. Therefore having those funded people operate
+in ways that attract more volunteers and enhance their participation experience
+is critical. That key principle motivates many of the more detailed principles
+given below for what to pay for and whom to pay.
+
+The approach for spending funds will be:
+
+- first figure out what we want to fund,
+- then look for a great candidate,
+- after that's settled, determine a fair compensation level.
+
+The next sections go into detail on each of these three points.
+
+What to pay for
+```````````````
+
+1. Pay for things that are important *and* otherwise won't get done.
+   *Rationale: there is way more to be done than there are funds to do all
+   those things. So count on interested volunteers or external sponsored work
+   to do many of those things.*
+2. Plan for sustainability. Don't rely on money always being there.
+3. Consider potential positive benefits for NumPy maintainers and contributors,
+   maintainers of other projects, end users, and other stakeholders like
+   packagers and educators.
+4. Think broadly. There's more to a project than code: websites, documentation,
+   community building, governance - it's all important.
+5. For proposed funded work, include paid time for others to review your work
+   if such review is expected to be significant effort - do not just increase
+   the load on volunteer maintainers.
+   *Rationale: we want the effect of spending funds to be positive for
+   everyone, not just for the people getting paid. This is also a matter of
+   fairness.*
+
+When considering development work, principle (1) implies that priority should
+be giving to (a) the most boring/painful tasks that no one likes doing, and to
+necessary structural changes to the code base that are too large to be done by
+a volunteer in a reasonable amount of time.
```

Comment:
This indeed can happen. The first principle in "what to pay for" says to pay for things that otherwise won't get done. One reason something doesn't get done is because the required skills are missing in the project/community.

I think we are missing skills, but more outside of development work. It's very well possible that it's a combination of a rare skill and no one in the team with bandwidth + motivation to address the need.

So we look outside of the community to find the right person. I think that's addressed in the next section ("who to pay").